openlcs/libs/kojiconnector.py

```diff
@@ -483,3 +489,51 @@ def download_source(self, build):
             err_msg = f'Failed to download source. Reason: {err}'
             raise RuntimeError(err_msg) from None
         return temp_dir
+
+    def get_module_mapping_srpms(self, module_nvr):
+        """
+        Return module SRPM NVRs.
+        """
+        build = self.get_build(module_nvr)
+        tag = build.get('extra').get('typeinfo').get('module').get(
+                'content_koji_tag')
+        try:
+            builds = self.list_tagged(tag)
+        except RuntimeError as err_msg:
+            err_msg = f'Failed to get the module {module_nvr} SRPM nvrs'
+            raise RuntimeError(err_msg) from None
+        package_nvrs = []
+        for build in builds:
+            package_nvrs.append(build.get('nvr'))
+        return package_nvrs
+
+    def get_module_component(self, module_nvr):
+        """
+        Return module component itself.
+        """
+        nvr = koji.parse_NVR(module_nvr)
+        module_component = [get_component_flat(nvr, 'RPMMOD')]
+        return module_component
+
+    def get_module_srpm_components(self, module_nvr):
+        """
+        Get module child srpm components.
+        """
+        srpm_nvrs = self.get_module_mapping_srpms(module_nvr)
+        srpm_components = []
+        for srpm_nvr in srpm_nvrs:
+            nvr = koji.parse_NVR(srpm_nvr)
+            srpm_components.append(get_component_flat(nvr, 'RPM'))
+        return srpm_components
+
+    def get_module_components(self, module_nvr):
```

Comment:
Functions like this should be outside of the lib, it's ok to keep them here for now IMO.

Reply:
Thanks. I planned to create a new file, like module_handlers.py, but I just thought the get_remote_source_components is in this module and this will call some functions of kojiconnector. So I kept them here :).